### PR TITLE
Compatible with django3 api + GH actions

### PR DESCRIPTION
**Pull request by gustavo-salazar**
This PR contains basically 2 changes:
1. in the `api.js` anytime it reads `someApiResponse.data.id` is changed to `someApiResponse.data && someApiResponse.data.id` because the API now returns `data: null` when is empty instead of `data: {data: null}`
2. Changing from travis to GH actions. Most of the changed files in the PR are basically moving them from `travis/` to `ci/`

*Note 1*: Sorry about all the prettier changes, I tried to control all the noise in the PR by adding a `.prettier` but it's still full of meaningless formatting changes.

*Note 2*: Please squash all the commits of this PR, as most of them are just trials into the GH actions workflow.

### CODE REVIEW

**Review by caballero — CHANGES_REQUESTED** [review]
Changes and additions are ok, however, I noticed some .DS_store files being part of the repo, as this push is moving the test files, could be good to delete those files. Also, there are some sha1 files for the sha1 files (ci/emg_api_datafiles/results/2017/11/ERP104174/version_5.0/ERZ477/006/ERZ477576_FASTA/hashes/ERZ477576_FASTA.fasta.gz.sha1.sha1)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/5349bdf1c0204b28940d3bf19c894d95)](https://www.codacy.com/app/mb1069/ebi-metagenomics-webkit?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=EBI-Metagenomics/ebi-metagenomics-webkit&amp;utm_campaign=Badge_Grade)
-[![Build Status](https://travis-ci.com/EBI-Metagenomics/ebi-metagenomics-webkit.svg?branch=develop)](https://travis-ci.com/EBI-Metagenomics/ebi-metagenomics-webkit)
 
-NPM package for a Backbone-JS API to consume the European Bioinformatic Institute's MGnify API for metagenomic data, in addition to a widget library to visualise the data
\ No newline at end of file
+[![Codacy Badge](https://api.codacy.com/project/badge/Coverage/5349bdf1c0204b28940d3bf19c894d95)](https://www.codacy.com/app/mb1069/ebi-metagenomics-webkit?utm_source=github.com&utm_medium=referral&utm_content=EBI-Metagenomics/ebi-metagenomics-webkit&utm_campaign=Badge_Coverage)
```
Comment (mberacochea) [inline README.md:3]: Codacy is still using Miguel's account, I've createa a ticket to migrate that

**Review by mberacochea — APPROVED** [review]
Thank you Gustavo, it looks OK to me. Only thing is to remove those .DS files.

I'll merge (and squash) this as soon as I can.

**No problem :)**
      Note 1: Sorry about all the prettier changes, I tried to control all the noise in the PR by adding a .prettier but it's still full of meaningless formatting changes.

**Yes, I'll squash and merge this.**
     Note 2: Please squash all the commits of this PR, as most of them are just trials into the GH     actions workflow.

**Comment by gustavo-salazar** [comment]
I have deleted the files requested by @caballero and also notice the tests were failing because of some changes in the API model; which is now fixed.

**Pull request merged**